run_test.sh

```diff
@@ -8,4 +8,9 @@ cargo run -p quick-protobuf --example pb_rs_example_v3_owned
 cargo run -p quick-protobuf --example pb_rs_example
 cargo run -p quick-protobuf --example pb_rs_example_v3
 
+# test that no_std can build
+pushd quick-protobuf/no-std-example
```

Comment:
Can we condense the example into a single file the rest of the examples? I assume the feature toggling makes it a little trickier... If it isn't too much work, it'd be nice to have it be consistent with the other examples.

Reply:
Done.